common/src/main/java/com/wynntils/screens/settings/widgets/TextInputBoxSettingsWidget.java

```diff
@@ -17,6 +20,7 @@ protected TextInputBoxSettingsWidget(Config<T> config, TextboxScreen textboxScre
         super(0, 6, width, FontRenderer.getInstance().getFont().lineHeight + 8, null, textboxScreen);
         this.config = config;
         setTextBoxInput(config.get().toString());
+        tooltip = ComponentUtils.wrapTooltips(List.of(Component.literal(config.getDescription())), 150);
```

Comment:
either this is not needed or this fixes something that you didnt describe..

Reply:
This sets the tooltip to be displayed, no?

It might be better to have the tooltip as a parameter in the parent constructor, then it's easier to set tooltips for any other input widgets

Reply:
The other change sets the tooltip to be rendered? this just sets some variable

Reply:
i would have to read the code, this is kind of weird

Reply:
Yeah this sets what the tooltip will be, in `TextInputBoxWidget` the variable is just null, it's only set here

Reply:
the tooltip variable refers to the field I added in `TextInputBoxWidget`. I did it like this to avoid another constructor parameter (because it is just used once) and to allow it to be changed.